.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Do we need this `.gitignore` or can we put this value in the global one?

Reply:
This is actually created by husky during install.  We can remove it from the global one if you would like, though it was put in place so that if you switch between branches that are and are not updated to husky 5, it remains gitignored.